src/domain/phone-provider/errors.ts

```diff
@@ -2,4 +2,5 @@ export class PhoneProviderServiceError extends Error {
   name = this.constructor.name
 }
 
+export class NotAValidPhoneNumberPhoneProviderError extends PhoneProviderServiceError {}
```

Comment:
imo 'invalid' would be a better prefix than 'notAValid'

Reply:
fixed